pkg/ctrlz/ctrlz.go

```diff
@@ -121,20 +121,20 @@ func RegisterTopic(t fw.Topic) {
 	allTopics = append(allTopics, t)
 }
 
-var listeningTestProbe func()
-
 // Run starts up the ControlZ listeners.
 //
 // ControlZ uses the set of standard core topics, the
 // supplied custom topics, as well as any topics registered
 // via the RegisterTopic function.
-func Run(o *Options, customTopics []fw.Topic) {
-	shutdown.Add(1)
-	defer shutdown.Done()
-
+func Run(o *Options, customTopics []fw.Topic) (*Server, error) {
 	if o.Port == 0 {
 		// disabled
-		return
+		s := &Server{}
+
+		s.shutdown.Add(1)
```

Comment:
Not sure what purpose this serves?

Reply:
This was leftover code, I removed it since it's no longer useful given the new structure.